server/local/ldap.yml

```diff
@@ -1,9 +1,9 @@
 ## Default values for anonymous access to UMich ldap.
 ## They can be overridden when a LdapCheck instance is constructed.
 #host: ldap.itd.umich.edu
-host: umod-test.mc.itd.umich.edu
+#host: umod-test.mc.itd.umich.edu
```

Comment:
Wonder whether the host switch can be done through some property file change -- no need to check in to github every time.